lib/connection.js

```diff
@@ -29,17 +29,19 @@ function Connection(options) {
   this.apiCallCount = 0;
 
   // Convenient methods, matches that of restler's
-  _.each(restlerMethodArgCount, function(count, method) {
+  _.forEach(restlerMethodArgCount, (count, method) => {
     this[method] = function(var_args) {
       var args = Array.prototype.slice.call(arguments);
       while (args.length < count) {
         args.push(undefined);
       }
       args.unshift(method);
-
       return this._request.apply(this, args);
     };
-  }, this);
+    this[method];
```

Comment:
Yeah, so [forEach in 4.17.11](https://lodash.com/docs/4.17.11#forEach) takes just two arguments, which was the major change from [forEach in 2.4.2](https://lodash.com/docs/2.4.2#forEach), which has three.

When I changed lodash initially, the tests broke because of "this._connection.get is not a function" because this piece wasn't working. So in this case, instead of this as a callback, I just called this[method] to get the rest methods into the connection object.